### ISSUE TITLE
Enhancement: Простой переход (ссылка) на официальную карточку ДТП

### ISSUE THREAD

**Issue by bender-spb** [tadata-ru/dtp-stat#21]
Предлагаю добавить возможность простого перехода с карточки ДТП проекта на карточку ДТП на официальном сайте.

![image](https://user-images.githubusercontent.com/9783949/52296367-d9ec2880-2986-11e9-90ef-a8b7fbbae67a.png)

* Это даст возможность удобнее валидировать данные конкретного ДТП сравнивая с официальным источником (видится полезным при работе с ДТП с неверными координатами)
*  Карточка ДТП проекта на данный момент не содержит всей информации карточки ДТП которая может быть полезна при таком анализе (например, километр дороги)
* Даже если карточка ДТП проекта будет отображать все имеющиеся данные - в дальнейшем такая фича может повысить прозрачность и создать большее доверие к проекту

**Comment by ustnv** [comment]
Не совсем понимаю, как это можно реализовать, так как внутри сервиса stat.gibdd.ru это делается дополнительными post запросами, абсолютных url-адресов на карточки нет.

**Comment by bender-spb** [comment]
Да, мне тоже не удалось пока увидеть чтобы это было реализуемо. 

Думаю что "Отображать больше информации на карточке ДТП" может закрыть частично эту потребность. 

Для валидации местоположения (при работе в рамках повышения качества данных) это как минимум отображать всегда:
- Координаты
- Адрес
- Дорога (трасса), километр дороги

**Comment by ustnv** [comment]
Можно сделать следующее:
- добавить возможность скачивания исходных json файлов (если несколько версий, то несколько с датами)
- добавить сообщение, если это ДТП было удалено со stat.gibdd.ru

**Comment by c13** [comment]
* добавить сообщение, если это ДТП было изменено в ежемесячном обновлении с сайта stat.gibdd.ru
